src/setup/sound.c

```diff
@@ -80,8 +80,7 @@ static char *gus_patch_path = NULL;
 static int gus_ram_kb = 1024;
 #ifdef _WIN32
 #define MAX_MIDI_DEVICES 20
-static char **midi_names;
-static int midi_num_devices;
+static char *midi_names[MAX_MIDI_DEVICES] = {"Microsoft MIDI Mapper"};
```

Comment:
This causes a constness problem. Is it possible to move initialization some place where you can duplicate the string to avoid that?